documentation/Installing-Clear-Containers-on-ClearLinux.md

```diff
@@ -12,20 +12,20 @@ $ sudo swupd update
 ## Install the Clear Containers bundle
 
 ```
-$ sudo swupd bundle-add containers-basic
+$ sudo swupd bundle-add containers-virt
 ```
 
 ## Restart the docker systemd service
 
-Docker on Clear Linux provides 2 service files to start the daemon, and only one will be enabled.
-- docker.service (runc)
-- docker-cor.service (cc-oci-runtime)
+Docker on Clear Linux provides `docker.service` service file to start the `docker` daemon.
+The daemon will use runc or cc-oci-runtime depending on the environment:
 
-If you are running Clear Linux on baremetal or on a VM with Nested Virtualization activated the service `docker-cor` will be launched and default runtime will be `cc-oci-runtime`. If you are running Clear Linux on a VM without Nested Virtualization the service `docker` will be launched and default runtime will be `runc`.
+
+If you are running Clear Linux on baremetal or on a VM with Nested Virtualization activated `docker` will use `cc-oci-runtime` as default runtime. If you are running Clear Linux on a VM without Nested Virtualization  `docker` use `runc` as default runtime.
+It is not need to register cc-oci-runtime manually, by default `docker` in Clear Linux will register atomatically.
```

Comment:
I think it would be clearer to say something like...

*It is not necessary to configure docker to use `cc-oci-runtime` manually since docker itself will automatically use this runtime on systems that support it.*

*Note: to check which runtime your system is using, run:*
```
sudo docker info | grep Runtime
```